client/blocks/reader-list-item/docs/example.jsx

```diff
@@ -0,0 +1,40 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import React, { PureComponent } from 'react';
+import { map } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import ConnectedReaderListItem from 'blocks/reader-list-item/connected';
+import ReaderListItemPlaceholder from 'blocks/reader-list-item/placeholder';
+import Card from 'components/card';
+
+const sites = {
+	longreads: { siteId: 70135762 },
+	wordpress: { feedId: 25823 },
+	bestBlogInTheWorldAAA: { siteId: 77147075 },
+	mathWithBadDrawings: { feedId: 10056049 },
+	uproxx: { feedId: 19850964 },
+	atlantic: { feedId: 49548095 },
+	fourthGenerationFarmGirl: { feedId: 24393283 },
+};
+
+export default class ReaderListItemExample extends PureComponent {
+	static displayName = 'ReaderListItem';
+
+	render() {
+		return (
+			<Card>
+				{ map( sites, site => (
+					<ConnectedReaderListItem key={ site.feedId || site.siteId } { ...site } />
+				) ) }
+				<ReaderListItemPlaceholder />
+			</Card>
+		);
+	}
+}
```

Comment:
Is this going to be included in DevDocs? I'm not sure it's necessary for it to be though.

<img width="783" alt="Screenshot 2019-07-06 at 09 31 37" src="https://user-images.githubusercontent.com/43215253/60753883-e1b10700-9fd0-11e9-9a09-c77e8480b470.png">

Reply:
> Is this going to be included in DevDocs? I'm not sure it's necessary for it to be though.

I think ideally we'd move any Reader-specific blocks out into `client/reader/blocks`, and out of Devdocs altogether, so probably best not to include this one. The example is still a useful addition :)